examples/binary-operators/binary_operators.metatags

```diff
@@ -0,0 +1,2 @@
+description: This BBE introduces the binary operators associated with ballerina types.
+keywords: ballerina, ballerina by example, bbe, operators, binary operators, bitwise operators, int, float, string, boolean, +, -, /, %, *, , <, >, <=, >=, >>, <<, >>>, ||, &&, |, &, ^
```

Comment:
IIRC, we used to have a limit on the number here. @sm1990 @keizer619 any concerns with these?

Reply:
That is not an issue anymore.